frontend/src/routes/cfp/form.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from "react";
+
+import { Button, InputField, SelectField, TextareaField } from "fannypack";
+import { Column, Row } from "grigliata";
+import * as yup from "yup";
+import { AUDIENCE_LEVEL_OPTIONS, BUTTON_PADDING, DURATION_OPTIONS, LANGUAGE_OPTIONS, ROW_PADDING, TOPIC_OPTIONS, TYPE_OPTIONS } from "./constants";
+
+const schema = yup.object().shape({
+  title: yup
+    .string()
+    .required()
+    .ensure(),
+  elevatorPitch: yup.string(),
+  abstract: yup
+    .string()
+    .required()
+    .ensure(),
+  topic: yup
+    .string()
+    .required()
+    .ensure(),
+  duration: yup
+    .string()
+    .required()
+    .ensure(),
+  language: yup
+    .string()
+    .required()
+    .ensure(),
+  type: yup
+    .string()
+    .required()
+    .ensure(),
+  audienceLevel: yup
+    .string()
+    .required()
+    .ensure(),
+});
+
+const InputWrapper: React.FC = props => {
+  return (
+    <Row paddingBottom={ROW_PADDING}>
+      <Column
+        columnWidth={{
+          mobile: 12,
+          tabletPortrait: 12,
+          tabletLandscape: 12,
+          desktop: 12,
+        }}
+      >
+        {props.children}
+      </Column>
+    </Row>
+  );
+};
+
+export const CFPForm = () => {
+  const [errors, setErrors] = useState({});
+  const [submission, setSubmission] = useState({
+    title: "",
+    abstract: "",
+    elevatorPitch: "",
+    topic: "",
+    duration: "",
+    language: "eng",
```

Comment:
The default value is a language is `en` if I remember correctly

Reply:
Yes, but here I am working with sample data, now I think I can do a Query with gatsby to retrieve constants, right??